src/sas/system/web.py

```diff
@@ -0,0 +1,24 @@
+class WebLinks:
+    def __init__(self):
+        self.nist_url = "https://www.nist.gov/"
+        self.umd_url = "https://www.umd.edu/"
+        self.sns_url = "https://neutrons.ornl.gov/"
+        self.nsf_url = "https://www.nsf.gov"
+        self.isis_url = "https://www.isis.stfc.ac.uk/"
+        self.ess_url = "https://europeanspallationsource.se/"
+        self.ill_url = "https://www.ill.eu/"
+        self.ansto_url = "https://www.ansto.gov.au/"
+        self.bam_url = "http://www.bam.de/"
+        self.delft_url = "http://www.tudelft.nl/en/tnw/business/facilities/reactor-instituut-delft/"
+        self.inst_url = "https://www.utk.edu"
+        self.diamond_url = "http://www.diamond.ac.uk"
+
+        self.homepage_url = "https://www.sasview.org"
+        self.download_url = 'https://github.com/SasView/sasview/releases'
+        self.marketplace_url = "http://marketplace.sasview.org/"
```

Comment:
If we can sort out sasview/sasmodel-marketplace#29 soon, this should become `https://marketplace.sasview.org/`

Reply:
ok, well you know where to change it when the time comes.